README.md

```diff
@@ -544,6 +544,7 @@ private fun startAuthoringPaymentActivity() {
             putExtra(EXTRA_AUTHORIZED_URLSTRING, authorizeUrl)
             putExtra(EXTRA_EXPECTED_RETURN_URLSTRING_PATTERNS, arrayOf(returnUrl))
             putExtra(EXTRA_UI_CUSTOMIZATION, uiCustomization)
+           // optional
```

Comment:
@AnasNaouchi WDYT about adding more context on why is this optional and when to use it?